python/ray/package.py

```diff
@@ -0,0 +1,90 @@
+import hashlib
+from pathlib import Path
+from zipfile import ZipFile
+
+from enum import Enum
+from ray.experimental import internal_kv
+
+PKG_DIR='/tmp'
+
+class Protocol(Enum):
+    GCS = "gcs://"
+    PIN_GCS = "pin_gcs://"
```

Comment:
What is the difference between GCS and PIN_GCS? Please also also leave a comment here for future readers.